listfilter.js

```diff
@@ -1,16 +1,105 @@
+            var obj2=list1[2];
+            var obj3=list1[3];
+            var obj4=list1[4];
+            var obj5=list1[5];
+            var obj6=list1[6];
+            var obj7=list1[7];
+            var obj8=list1[8];
+            var obj9=list1[9];
+                document.getElementById("1").innerHTML = "Id:"+obj.Id+",name:"+obj.Name+",color:"+obj.color+",Price:"+obj.Price+",company:"+obj.Company+",Memory:"+obj.Memory+",feature:camera:"+obj.feature.camera+",sensor:"+obj.feature.sensor+",touch:"+obj.feature.touch;
+                document.getElementById("2").innerHTML = "Id:"+obj1.Id+",name:"+obj1.Name+",color:"+obj1.color+",Price:"+obj1.Price+",company:"+obj1.Company+",Memory:"+obj1.Memory+",feature:camera:"+obj1.feature.camera+",touch:"+obj1.feature.touch;
+                document.getElementById("3").innerHTML = "Id:"+obj2.Id+",name:"+obj2.Name+",color:"+obj2.color+",Price:"+obj2.Price+",company:"+obj2.Company+",Memory:"+obj2.Memory+",feature:camera:"+obj2.feature.camera;
+                document.getElementById("4").innerHTML = "Id:"+obj3.Id+",name:"+obj3.Name+",color:"+obj3.color+",Price:"+obj3.Price+",company:"+obj3.Company+",Memory:"+obj3.Memory+",feature:camera:"+obj3.feature.camera+",sensor:"+obj3.feature.sensor+",touch:"+obj3.feature.touch;
+                document.getElementById("5").innerHTML = "Id:"+obj4.Id+",name:"+obj4.Name+",color:"+obj4.color+",Price:"+obj4.Price+",company:"+obj4.Company+",Memory:"+obj4.Memory+",feature:camera:"+obj4.feature.camera+",touch:"+obj4.feature.touch;
+                document.getElementById("6").innerHTML = "Id:"+obj5.Id+",name:"+obj5.Name+",color:"+obj5.color+",Price:"+obj5.Price+",company:"+obj5.Company+",Memory:"+obj5.Memory+",feature:camera:"+obj5.feature.camera+",sensor:"+obj5.feature.sensor+",touch:"+obj5.feature.touch;
+                document.getElementById("7").innerHTML = "Id:"+obj6.Id+",name:"+obj6.Name+",color:"+obj6.color+",Price:"+obj6.Price+",company:"+obj6.Company+",Memory:"+obj6.Memory+",feature:camera:"+obj6.feature.camera+",touch:"+obj6.feature.touch;
+                document.getElementById("8").innerHTML = "Id:"+obj7.Id+",name:"+obj7.Name+",color:"+obj7.color+",Price:"+obj7.Price+",company:"+obj7.Company+",Memory:"+obj7.Memory+",feature:camera:"+obj7.feature.camera+",sensor:"+obj7.feature.sensor+",touch:"+obj7.feature.touch;
+                document.getElementById("9").innerHTML = "Id:"+obj8.Id+",name:"+obj8.Name+",color:"+obj8.color+",Price:"+obj8.Price+",company:"+obj8.Company+",Memory:"+obj8.Memory+",feature:camera:"+obj8.feature.camera+",touch:"+obj8.feature.touch;
+                document.getElementById("10").innerHTML ="Id:"+obj9.Id+",name:"+obj9.Name+",color:"+obj9.color+",Price:"+obj9.Price+",company:"+obj9.Company+",Memory:"+obj9.Memory+",feature:camera:"+obj9.feature.camera;
+            });
+        } 
 
+    function showLaptopData(){
+        $("#l1").click(function(){
+            $("#hello12").show();
+            var ob=list2[0];
```

Comment:
Put all this in loop. [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself).